aws-lambda-layer/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -x
+
+set -e
+
+FREETDS_VERSION=${FREETDS_VERSION:=1.1.6}
+IMAGE_NAME=veracross/freetds-layer:${FREETDS_VERSION:latest}
+
+rm -rf ./build
+mkdir ./build
+
+docker build \
```

Comment:
should there also be a `docker push` step in here?

Reply:
Probably - I'll add the step.